crates/leon/src/template.rs

```diff
@@ -0,0 +1,137 @@
+use std::{borrow::Cow, io::Write, ops::Add};
```

Comment:
I'd prefer to use beef::Cow as it is 8 bytes smaller.

We already use beef in binstall-manifest, so using it here would not add more dep.

Reply:
In that case, Literal needs to be a newtype instead a type alias since the API of beef::Cow is slightly different from StdCow: It is not an enum, does not have Borrowed and Owned variant, but instead of associated fn borrowed and owned.